test/cpp/interop/xds_interop_client.cc

```diff
@@ -123,7 +142,9 @@ class TestClient {
   TestClient(const std::shared_ptr<Channel>& channel)
       : stub_(TestService::NewStub(channel)) {}
 
-  void AsyncUnaryCall() {
+  void AsyncUnaryCall(
+      absl::optional<std::vector<std::pair<std::string, std::string>>>
```

Comment:
Why use `absl::optional<>` here?  Why not just pass in an empty map if there's no metadata?